backend/app/routes/recommend.py

```diff
@@ -1,30 +1,28 @@
 """The Recommend endpoint"""
-from datetime import datetime
 from json import JSONDecodeError
 
 from fastapi import Body, HTTPException, Query
 from httpx import AsyncClient
-from pydantic import BaseModel
 
-from app.config import RS_ROWS, RS_URL, SOLR_URL
+from app.config import RS_ROWS
+from app.recommender.operations import recommendations
+from app.requests import RecommendRequest
+from app.solr.operations import search
 
 from .util import DEFAULT_SORT, internal_api_router
 
 
-class RecommendRequest(BaseModel):
-    """POST /recommend request"""
-
-    unique_id: str
-    timestamp: datetime
-    visit_id: str
-    page_id: str
-    panel_id: str
-
-
+# pylint: disable=too-many-arguments
 @internal_api_router.post("/recommend")
 async def recommend_post(
-    q: str = Query(..., description="Query string"),
     collection: str = Query(..., description="Collection"),
+    q: str = Query(..., description="Query string"),
```

Comment:
This same as above. Docstrings with params or URL to its documentations + examples